dask_kubernetes/operator/controller/controller.py

```diff
@@ -631,8 +632,12 @@ async def daskworkergroup_replica_update(
                     annotations.update(**worker_spec["metadata"]["annotations"])
                 if "labels" in worker_spec["metadata"]:
                     labels.update(**worker_spec["metadata"]["labels"])
+
+            SIZE = dask.config.get("kubernetes.controller.worker-allocation.batch-size")
+            DELAY = dask.config.get("kubernetes.controller.worker-allocation.delay")
+            batch_size = min(workers_needed, SIZE)
```

Comment:
This will raise an exception if `SIZE` is `None` (the default)
```suggestion
            batch_size = min(workers_needed, SIZE) if SIZE else workers_needed
```